composer.json

```
@@ -1,6 +1,6 @@
 {
     "require": {
-        "mobbexco/php-plugins-sdk": "1.15.0",
+        "mobbexco/php-plugins-sdk": "1.15.1",
```

En la 1.15.2 se elimina el prefijo de los planes:

```suggestion
        "mobbexco/php-plugins-sdk": "1.15.2",
```